### DISCUSSION TITLE
Non Latin symbols in text from form encoded like in HTML/XML

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#6885] [Q&A]
Example  
```
The following information was submitted:

Name: \u0414\u0438\u043C\u0430

Email: nde96m@gmail.com

Phone: 6586485684 
``` 

How can I decode them back or force them not encoding?
like `{{ Request.Form.Name | url_decode }}` does.

**Comment by Skrypt** [comment]
Razor : 

`System.Net.WebUtility.HtmlDecode(string)`

Liquid : 

https://docs.orchardcore.net/en/dev/docs/reference/modules/Liquid/#html-escaping

Also should investigate if https://shopify.github.io/liquid/tags/raw/ would be approriate to implement if not.

**Comment by deanmarcussen** [comment]
Or customise your encoder. https://docs.orchardcore.net/en/dev/docs/guides/encoding-settings/

We introduced DI encoders for all workflows to resolve this.

**Reply by hishamco** [reply]
I think @sebastienros inject the `HtmlEncoder` through DI long time back
